recipes/yaml/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.1.6" %}
+
+package:
+  name: yaml
+  version: {{ version }}
+
+source:
+  fn: yaml-{{ version }}.tar.gz
+  url: http://pyyaml.org/download/libyaml/yaml-{{ version }}.tar.gz
+  md5: 5fe00cda18ca5daeb43762b80c38e06e
+  patches:
+    # Change CMakeLists so that we can build static and shared libraries.
+    # See this PR ( https://github.com/yaml/libyaml/pull/10 ) for details.
+    - CMakeLists.txt.patch
```

Comment:
This patch ( https://github.com/yaml/libyaml/pull/10 ) let's us build DLLs and LIBs. It was the simplest thing I could come up with. Though improvements are certainly welcome.